marshal_test.go

```diff
@@ -1329,7 +1373,7 @@ func opaqueDoubleResponse() []byte {
 	return []byte{
 		0x30, 0x38, 0x02, 0x01, 0x01, 0x04, 0x06, 0x70, 0x75, 0x62, 0x6c, 0x69,
 		0x63, 0xa2, 0x2b, 0x02, 0x04, 0x23, 0xd5, 0xd7, 0x05, 0x02, 0x01, 0x00,
-		0x02, 0x01, 0x00, 0x30, 0x1d, 0x30, 0x17, 0x06, 0x0c, 0x2b, 0x06, 0x01,
+		0x02, 0x01, 0x00, 0x30, 0x1d, 0x30, 0x1b, 0x06, 0x0c, 0x2b, 0x06, 0x01,
```

Comment:
Can you elaborate this particular change a bit more?

Reply:
This is the full size of a single variable binding. In this case, a size of 23 bytes (0x17) was specified, but the actual size is 27 bytes (0x1b).
The comment says that this set was manually compiled as described in the description, so I think it was this error.
During unmarshaling, this field is currently ignored, so no errors are thrown.

I am working on a parsing fix to account for this length. I think I will issue a separate PR.

Reply:
Clear, thank you.